frontend/config/webpack.local.hmr.config.js

```diff
@@ -7,6 +7,7 @@ module.exports = (opts) => {
 
   return {
     ...config,
+    devtool: 'source-map',
```

Comment:
This change should not be necessary, we are extending the webpack.local.config which already has the same line in it